transformations/americanize_britishize_english/transformation.py

```diff
@@ -0,0 +1,131 @@
+        super().__init__(seed, max_outputs=max_outputs)
+
+        # Creating a custom vocab dictionary consisting of totally different words for same context
+        difference_british_to_american = {
+            "trousers": "pants",
+            "flat": "apartment",
+            "bonnet": "hood",
+            "boot": "trunk",
+            "lorry": "truck",
+            "university": "college",
+            "holiday": "vacation",
+            "jumper": "sweater",
+            "trainers": "sneakers",
+            "postbox": "mailbox",
+            "biscuit": "cookie",
+            "chemist": "drugstore",
+            "shop": "store",
+            "football": "soccer",
+            "autumn": "fall",
+            "barrister": "attorney",
+            "bill": "check",
+            "caravan": "trailer",
+            "cupboard": "closet",
+            "diversion": "detour",
+            "dustbin": "trash can",
+            "jug": "pitcher",
+            "lift": "elevator",
+            "mad": "crazy",
+            "maize": "corn",
+            "maths": "math",
+            "motorbike": "motorcycle",
+            "motorway": "freeway",
+            "nappy": "diaper",
+            "pavement": "sidewalk",
+            "post": "mail",
+            "postman": "mailman",
+            "pub": "bar",
+            "rubber": "eraser",
+            "solicitor": "attorney",
+            "tax": "cab",
+            "timetable": "schedule",
+            "torch": "flashlight",
+            "waistcoat": "vest",
+            "windscreen": "windshield",
+            "angry": "mad",
+            "caretaker": "janitor",
+            "cot": "crib",
+            "curtains": "drapes",
+            "engine": "motor",
+            "garden": "yard",
+            "handbag": "purse",
+            "hoarding": "billboard",
+            "ill": "sick",
+            "interval": "intermission",
+            "luggage": "baggage",
+            "nowhere": "noplace",
+            "optician": "optometrist",
+            "queue": "line",
+            "rubbish": "trash",
+        }
+        # Replacing the keys with values and vice versa for the custom vocab dictionary
+        # And merging both of them
+        vocab_diff = dict(
+            (v, k) for k, v in difference_british_to_american.items()
+        )
+        vocab_diff.update(difference_british_to_american)
+
+        """
+         Gets the british to american english dictionary
+         Gets the american to British english dictionary
+         Merges both these dictionaries with the custom vocab dictionary.
+        """
+        url = "https://raw.githubusercontent.com/hyperreality/American-British-English-Translator/master/data/american_spellings.json"
+        american_british_dict = requests.get(url).json()
+
+        url = "https://raw.githubusercontent.com/hyperreality/American-British-English-Translator/master/data/british_spellings.json"
+        british_american_dict = requests.get(url).json()
+
```

Comment:
What will happen if someone will try to run this transformation offline? 
Would it cast an error? If so should it be handled?

Reply:
Agree with this problem ^.
I recommend just downloading it once, and then including 2 jsons in the directory. It should be pretty small.
But at least having error handling would be good.

Reply:
There is nothing like an offline transformation these days so I assumed this approach should be fine. But based on your inputs I can make the changes. What should be the right implementation? 
For now, I have added the error handling. Let me know if more changes are required. Thanks

Reply:
Thanks for adding the error handling. I agree that it least safety thing but I am in favour of @RoyRin proposal to add jsons to the directory

Reply:
Sure I can add it in later increments. Thanks